### PR TITLE
fix: fix directoryWatcher change event content

### PR DESCRIPTION
**Pull request by DiamondYuan**

### CODE REVIEW

File: lib/DirectoryWatcher.js [hunk 1]
```diff
@@ -235,7 +235,7 @@ DirectoryWatcher.prototype.watch = function watch(filePath, startTime) {
 		if(data) {
 			var ts = data[0] === data[1] ? data[0] + FS_ACCURACY : data[0];
 			if(ts >= startTime)
-				watcher.emit("change", data[1]);
+				watcher.emit("change", filePath, data[1]);
```
Comment (sokra) [inline lib/DirectoryWatcher.js:238]: Not sure, but my first guess is this should depend on `filePath === this.path`:

```suggestion
				if(filePath === this.path)
					watcher.emit("change", data[1]);
				else
					watcher.emit("change", filePath, data[1]);
```

Comment (DiamondYuan) [reply lib/DirectoryWatcher.js:238]: arguments of “change" event can it be one ?

https://github.com/webpack/watchpack/blob/6758e8e109e1d1b6d2042397f80f68c7a78b9107/lib/watchpack.js#L98

```
Watchpack.prototype._fileWatcher = function _fileWatcher(file, watcher) {
	watcher.on("change", function(mtime, type) {
		this._onChange(file, mtime, file, type);
	}.bind(this));
	watcher.on("remove", function(type) {
		this._onRemove(file, file, type);
	}.bind(this));
	return watcher;
};

```

Comment (DiamondYuan) [reply lib/DirectoryWatcher.js:238]: i will add test.Maybe you are right.

**Comment by alexander-akait** [comment]
Closing due to inactivity. Thanks!

**Pull request closed without merging**